docs/2_2_common_schema.adoc

```diff
@@ -1032,14 +1081,14 @@ Even if this Dirac impulse would be modeled correctly by the modeling environmen
 Furthermore, in many cases the model equations are derived under the assumption of a constant value (like mass or capacity), and the model equations would be different if "p" would be time varying._
 
 _FMI for Model Exchange:_ +
-_Therefore, "tuning a parameter" during simulation does not mean to "change the parameter online" during simulation.
+_Therefore, "tuning a (structural) parameter" during simulation does not mean to "change the parameter online" during simulation.
 Instead, this is a short hand notation for:_
 
 . _Stop the simulation at an event instant (usually, a step event, in other words, after a successful integration step)._
 
-. _Change the values of the tunable parameters._
+. _Change the values of the tunable (structural) parameters. For tunable structural parameters, the "reconfiguration Mode" state must be entered before and left afterwards._
```

Comment:
Are we using double quotes for states? If so this should go in the contribution guide.

Reply:
In other sections, we write the states without quotes but in bold letters. (I will fix it)

General topic:
We should try to avoid state names that consist of more than 1 word.
Additionally, we should either call it mode or state. Currently this is quite messy.

Reply:
Actually, my last comment is unfortunately not true: State names are not consistently written in bold in other sections. (sometimes bold, sometimes not)

I suggest to write state names bold.